### PR TITLE
Clarify when current active Span is not used as a parent for new Spans

### PR DESCRIPTION
**Pull request by tigrannajaryan**

### CODE REVIEW

File: specification/trace/api.md [hunk 1]
```diff
@@ -136,9 +136,11 @@ with an invalid `SpanContext` if there is no currently active `Span`.
...
 When creating a new `Span`, the `Tracer` MUST allow the caller to specify the
 new `Span`'s parent in the form of a `Span` or `SpanContext`. The `Tracer`
-SHOULD create each new `Span` as a child of its active `Span` unless an
-explicit parent is provided or the option to create a span without a parent is
-selected, or the current active `Span` is invalid.
+SHOULD create each new `Span` as a child of its active `Span` unless one of the
+following is true:
+- an explicit parent is provided,
+- the option to create a span without a parent is selected,
+- the current active `Span` is invalid.
```
Comment (Oberon00) [inline specification/trace/api.md:143]: There is no such thing as an invalid Span, see https://github.com/open-telemetry/opentelemetry-specification/pull/619#discussion_r434951828

Comment (yurishkuro) [reply specification/trace/api.md:143]: +1, otherwise a good change/clarification. I like bullet points :-)

File: specification/trace/api.md [hunk 2]
```diff
@@ -136,9 +136,11 @@ with an invalid `SpanContext` if there is no currently active `Span`.
 
 When creating a new `Span`, the `Tracer` MUST allow the caller to specify the
 new `Span`'s parent in the form of a `Span` or `SpanContext`. The `Tracer`
-SHOULD create each new `Span` as a child of its active `Span` unless an
-explicit parent is provided or the option to create a span without a parent is
-selected, or the current active `Span` is invalid.
+SHOULD create each new `Span` as a child of its active `Span` unless one of the
```
Comment (arminru) [inline specification/trace/api.md:139]: Is this a SHOULD or should it rather be a MUST?

Comment (Oberon00) [inline specification/trace/api.md:139]: Not directly related, but while you are changing this area:

```suggestion
SHOULD create each new `Span` as a child of the active `Span` unless one of the
```

As #619 clarifies, there is no per-tracer active span.

Comment (Oberon00) [reply specification/trace/api.md:139]: I think the spec says that you don't need to maintain a currently active span at all (at least not without an explicit Context argument passed to each function). But I agree that if an active Span exists, it MUST be used IMHO.

File: specification/trace/api.md (hunk 1 above)
Comment (tigrannajaryan) [reply specification/trace/api.md]: Removed.

**Comment by tigrannajaryan** [comment]
Closing, this was overwritten / clarified by another PR.

**Pull request closed without merging**